components/raftstore/src/store/fsm/apply.rs

```diff
@@ -837,8 +845,7 @@ where
 
             let res = match entry.get_entry_type() {
                 EntryType::EntryNormal => self.handle_raft_entry_normal(apply_ctx, &entry),
-                EntryType::EntryConfChange => self.handle_raft_entry_conf_change(apply_ctx, &entry),
-                EntryType::EntryConfChangeV2 => unimplemented!(),
+                t => self.handle_raft_entry_conf_change(apply_ctx, &entry, t),
```

Comment:
I suggest to clarify exact entry types instead to detect entry type changes in the future.